packages/components/tag/src/tag.vue

```diff
@@ -12,7 +12,12 @@
       <Close />
     </el-icon>
   </span>
-  <transition v-else :name="`${ns.namespace.value}-zoom-in-center`" appear>
+  <transition
+    v-else
+    :name="`${ns.namespace.value}-zoom-in-center`"
+    appear
+    @vnode-mounted="handleVNodeMounted"
```

Comment:
```suggestion
    @vue:mounted="handleVNodeMounted"
```

![image](https://github.com/user-attachments/assets/66acc359-119e-4d61-9af2-e5c7de7dcbe7)